src/vsr/superblock_free_set.zig

```diff
@@ -578,12 +651,15 @@ test "FreeSet checkpoint" {
     const expectEqual = std.testing.expectEqual;
     const blocks_count = FreeSet.shard_bits;
     var set = try FreeSet.init(std.testing.allocator, blocks_count);
+    set.opened = true;
```

Comment:
It is tempting to say that the free set is opened after init (as it indeed is in a valid state where no blocks are allocated), but I'd rather not do it -- while it is a valid state, if we actually do have a non-empty free set on disk, _using_ that valid state can lose data!

Reply:
💯 critical.